mava/systems/tf/madqn/networks.py

```diff
@@ -61,13 +60,20 @@ def make_default_networks(
         Mapping[str, types.TensorTransformation]: returned agent networks.
     """
 
-    # Set Policy function and layer size
+    # Set Policy function and layer size.
+    # Default size per arch type.
     if archecture_type == ArchitectureType.feedforward:
+        if not policy_networks_layer_sizes:
```

Comment:
Nice I like that we can now have different defaults for feedforward and recurrent.